spikeinterface/core/segmentutils.py

```diff
@@ -218,6 +217,54 @@ def concatenate_recordings(*args, **kwargs):
 concatenate_recordings.__doc__ == ConcatenateSegmentRecording.__doc__
 
 
+class SplitSegmentRecording(BaseRecording):
```

Comment:
I think we should have somethin that select one or more segment and call this SelectSegmentRecroding.

than we could have 2 function on top of it : `split_segment` (or split_segment_recording) and `select_some_segment()`

Theses 2 could also be avialable directly with a method I think

Reply:
ok!